app.js

```diff
@@ -48,166 +52,162 @@ function wantsToPlay(){
     console.log('"wantsToPlay" = ' + wantToPlay + '. continuing quiz');
     return true;
   }
+}
+//start of questions
```

Comment:
This is literally what the following line does. Probably not a comment worth keeping, once you're more comfortable.